helmfile.d/config/otv-backend.yaml.gotmpl

```diff
@@ -14,10 +14,10 @@ image:
 
```

Comment:
as you can see, this configuration applies to both kusama and polkadot, since it is in /config

What I'd suggest instead is to apply this modification to Kusama only => /config/kusama/otv-backend.yaml.gotmpl